app/ui-react/syndesis/src/modules/data/pages/viewEditor/ViewEditorSqlPage.tsx

```diff
@@ -201,6 +203,7 @@ export const ViewEditorSqlPage: React.FunctionComponent = () => {
           0
         );
       }
+      setIsLoadingPreview(false);
```

Comment:
Add a "setIsLoadingPreview(false)" in the catch block, so the spinner will dismiss when there is an error condition

Reply:
Done